### PR TITLE
LPS-143217 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-143217

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/basiccomponents/BasicComponents.testcase [hunk 1]
```diff
@@ -400,7 +400,7 @@ definition {
 		}
 	}
 
-	@description = "This is a test for LPS-98241 and LPS-128667. The user could configure the Paragraph fragment in page editor."
+	@description = "This is a test for LPS-98241, LPS-98753 and LPS-128667. The user could configure and duplicate the Paragraph fragment in page editor."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/basiccomponents/BasicComponents.testcase:403]: Add a new case in FragmentsUI.testcase
The step should be
Add a Heading fragment to a content page
Change the text color
Change the Heading Level
Edit the inline text of Heading
Duplicate the Heading
View the fragments styles are copied to duplicated one
View the edited inline text is copied to duplicated one

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/controlmenu/productmenu/ProductMenu.testcase [hunk 2]
```diff
@@ -18,7 +24,30 @@ definition {
...
 		}
 		else {
-			Site.tearDownCP();
+			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+		}
+	}
+
+	@description = "This is a test case for LPS-112994. The Product Menu icon should not be shown when the user is in the Control Panel Apps even though enable Live"
+	@priority = "4"
+	test NotVisibleInControlPanelAppsForStagedSite {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/controlmenu/productmenu/ProductMenu.testcase:33]: ViewProductMenuIconInvisibleInGlobalAppAfterEnableLive
Move case to ProductMenuWithStaging.testcase

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/controlmenu/productmenu/ProductMenu.testcase [hunk 3]
```diff
@@ -18,7 +24,30 @@ definition {
...
+		task ("Enable local staging") {
+			JSONStaging.enableLocalStaging(groupName = "Test Site Name");
+		}
+
+		task ("View product menu icon is shown in Pages admin") {
+			SitePages.openPagesAdmin(siteURLKey = "test-site-name-staging");
+
+			AssertVisible(locator1 = "ProductMenu#TOGGLE");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/controlmenu/productmenu/ProductMenu.testcase:41]: No needed

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you update according to comments? Thanks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/basiccomponents/BasicComponents.testcase (hunk 1 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/basiccomponents/BasicComponents.testcase:403]: Thanks Tim! I'll update the PR.

**Comment by ozysouza** [comment]
> Hi @ozysouza, could you update according to comments? Thanks

Hi @Tim-Cao, PR updated following your comments, could you take a look at? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsUI.testcase [hunk 4]
```diff
@@ -75,6 +75,66 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-98753. Duplicating a fragmentEntryLink should copy the exact fragment configuration of the one that is copied."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsUI.testcase:78]: The fragment styles and inline text should be copied to duplicated one.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsUI.testcase [hunk 5]
```diff
@@ -75,6 +75,66 @@ definition {
...
+	@priority = "5"
+	test ViewDuplicatedFragmentConfiguration {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsUI.testcase:80]: DuplicateFragmentWithStylesAndInlineText

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsUI.testcase [hunk 6]
```diff
@@ -75,6 +75,66 @@ definition {
...
+
+		task ("Duplicate the Heading fragment") {
+			PageEditor.gotoDuplicate(fragmentName = "Heading");
+		}
+
+		task ("View duplicated Heading fragment") {
+			PageEditor.viewDuplicateFragment(
+				fragmentName = "Heading",
+				id = "Heading");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsUI.testcase:121]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsUI.testcase [hunk 7]
```diff
@@ -75,6 +75,66 @@ definition {
...
+
+		task ("View custom configurations are shown in duplicated Heading fragment") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsUI.testcase:123]: View custom fragment styles and edited inline text are copied to duplicated Heading

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update according to comments? Thanks

Hi @Tim-Cao, sure thing. PR updated.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/043bc123d1b0e20770df5546334a87a3915c506d...44d74b110bf0303b4ad797fcfb32faeaf84cf57d

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**